src/net/sourceforge/kolmafia/maximizer/Evaluator.java

```diff
@@ -67,7 +67,9 @@
   private boolean retroCapeNeeded = false;
   private boolean backupCameraNeeded = false;
 
+  /** Whether or not we should consider or not consider certain slots when maximizing */
```

Comment:
I don't understand the comment,

Reply:
It's a note for how `slots` works, because I found it confusing when reading through the file. It's a little bit of documentation.

To give an example, if `this.slots[EquipmentManager.HAT]` is non-negative, hats are considered, and if negative, hats aren't considered.

Reply:
How about 

if slots[i] > 0 then equipment of type i can be considered for maximization

That is much clearer to me and since you were confused enough to comment it seems reasonable to have a comment that informs both of us ;-)

Reply:
Clarified to:

> /** if slots[i] >= 0 then equipment of type i can be considered for maximization */